mergin/test/test_client.py

```diff
@@ -786,18 +798,27 @@ def test_available_storage_validation(mcStorage):
     try:
         mcStorage.push_project(project_dir)
     except ClientError as e:
+        print("Pushe fail")
         # Expecting "You have reached a data limit" 400 server error msg.
         assert "You have reached a data limit" in str(e)
         got_right_err = True
-    assert got_right_err
+    finally:
+        assert got_right_err
 
-    # Expecting empty project
-    project_info = get_project_info(mcStorage, STORAGE_WORKSPACE, test_project)
-    assert project_info["version"] == "v0"
-    assert project_info["disk_usage"] == 0
+        # Expecting empty project
+        project_info = get_project_info(mcStorage, STORAGE_WORKSPACE, test_project)
+        assert project_info["version"] == "v0"
+        assert project_info["disk_usage"] == 0
 
-    # remove dummy big file from a disk
-    remove_folders([project_dir])
+        # remove dummy big file from a disk
+        remove_folders([project_dir])
+
+        # revert storage limit to default value
```

Comment:
Maybe we should handle the creation and reset in the mcStorage fixture, but it's fine for now I think